python/cudf/cudf/_lib/cpp/lists/concatenate_rows.pxd

```diff
@@ -5,7 +5,7 @@ from libcpp.memory cimport unique_ptr
 from cudf._lib.cpp.column.column cimport column
 from cudf._lib.cpp.table.table_view cimport table_view
 
-cdef extern from "cudf/lists/concatenate_rows.hpp" namespace \
+cdef extern from "cudf/lists/combine.hpp" namespace \
```

Comment:
This file would likely have to be renamed too to `combine.pxd`

Reply:
Thanks. Done.